### ISSUE TITLE
Autofill lat/lon fails on ISO installs

### ISSUE THREAD

**Issue by apertome** [perfsonar/toolkit#231]
This works on all the RPM-installed hosts I've tried but in the ISOs the autofill lat/lon button never shows up:

/toolkit/auth/admin/

This error shows up in /var/log/httpd/ssl_error_log:

```
[Tue Apr 18 10:03:31.956131 2017] [cgi:error] [pid 13088] [client 129.79.9.117:60319] AH01215: Error Opening file /usr/share/GeoIP/GeoIPCity.dat, referer: https://129.79.9.164/toolkit/auth/admin/
[Tue Apr 18 10:03:32.019200 2017] [cgi:error] [pid 13088] [client 129.79.9.117:60319] AH01215: Can't call method "record_by_addr" on an undefined value at /usr/lib/perfsonar/web-ng/root/admin/services/../../../../lib/perfSONAR_PS/Utils/GeoLookup.pm line 105., referer: https://129.79.9.164/toolkit/auth/admin/
[Tue Apr 18 10:03:32.019254 2017] [cgi:error] [pid 13088] [client 129.79.9.117:60319] End of script output before headers: host.cgi, referer: https://129.79.9.164/toolkit/auth/admin/
```

**Comment by apertome** [comment]
I'm seeing something kind of odd here. With the ISO installs (tested with a full install), the GeoIP-data-1.5.0-11.el7.noarch package is not getting installed correctly. It installs the files, but does not create all the symlinks it's supposed to have.

Here are the files it installed:
-rw-r--r--. 1 root root  3730461 Nov  4 12:50 GeoIPASNum-initial.dat
-rw-r--r--. 1 root root  4675099 Nov  4 12:50 GeoIPASNumv6-initial.dat
-rw-r--r--. 1 root root 22255110 Nov  4 12:51 GeoIPCity-initial.dat
-rw-r--r--. 1 root root 23036134 Nov  4 12:51 GeoIPCityv6-initial.dat
lrwxrwxrwx. 1 root root       17 Apr 17 14:27 GeoIP.dat -> GeoIP-initial.dat
-rw-r--r--. 1 root root   890597 Nov  4 12:50 GeoIP-initial.dat
lrwxrwxrwx. 1 root root       19 Apr 17 14:27 GeoIPv6.dat -> GeoIPv6-initial.dat
-rw-r--r--. 1 root root  1667515 Nov  4 12:50 GeoIPv6-initial.dat``
After a `yum reinstall GeoIP-data.noarch` the additional symlinks are created:

`lrwxrwxrwx  1 root root       39 Apr 27 11:39 GeoIPASNum.dat -> /usr/share/GeoIP/GeoIPASNum-initial.dat
-rw-r--r--  1 root root  3730461 Nov  4 12:50 GeoIPASNum-initial.dat
lrwxrwxrwx  1 root root       41 Apr 27 11:39 GeoIPASNumv6.dat -> /usr/share/GeoIP/GeoIPASNumv6-initial.dat
-rw-r--r--  1 root root  4675099 Nov  4 12:50 GeoIPASNumv6-initial.dat
lrwxrwxrwx  1 root root       38 Apr 27 11:39 GeoIPCity.dat -> /usr/share/GeoIP/GeoIPCity-initial.dat
-rw-r--r--  1 root root 22255110 Nov  4 12:51 GeoIPCity-initial.dat
lrwxrwxrwx  1 root root       40 Apr 27 11:39 GeoIPCityv6.dat -> /usr/share/GeoIP/GeoIPCityv6-initial.dat
-rw-r--r--  1 root root 23036134 Nov  4 12:51 GeoIPCityv6-initial.dat
lrwxrwxrwx. 1 root root       17 Apr 17 14:27 GeoIP.dat -> GeoIP-initial.dat
-rw-r--r--. 1 root root   890597 Nov  4 12:50 GeoIP-initial.dat
lrwxrwxrwx. 1 root root       19 Apr 17 14:27 GeoIPv6.dat -> GeoIPv6-initial.dat
-rw-r--r--. 1 root root  1667515 Nov  4 12:50 GeoIPv6-initial.dat`


I can work around this issue by falling back to another filename, but I wonder why this is happening. @arlake228 any thoughts? I'm assuming maybe there's a post-install script that's not getting run when installing from ISOs, but I'm not sure why that would be.

**Comment by arlake228** [comment]
Does a `yum reinstall` of the package fix things? You could also dig in the anaconda logs under /var/log/anaconda and grep for anything about GeoIP-data. It should have any errors thrown by the RPM while the package was installing.

**Comment by apertome** [comment]
Yes, `yum reinstalll` fixes it. I don't see any errors when it installed (here is the log if it helps). 

```
# grep GeoIP-data /var/log/anaconda/*
/var/log/anaconda/journal.log:Apr 17 18:23:24 129-79-9-73.dhcp-bl.indiana.edu yum[1284]: TSINFO: Marking GeoIP-data-1.5.0-11.el7.noarch as install for libperfsonar-perl-4.0-1.el7.centos.noarch
/var/log/anaconda/journal.log:Apr 17 18:25:39 129-79-9-73.dhcp-bl.indiana.edu yum[1284]: TSINFO: Marking GeoIP-data-1.5.0-11.el7.noarch as install for libperfsonar-perl-4.0-1.el7.centos.noarch
/var/log/anaconda/journal.log:Apr 17 18:25:48 129-79-9-73.dhcp-bl.indiana.edu packaging[1284]: Member: GeoIP-data.noarch 0:1.5.0-11.el7 - u
/var/log/anaconda/journal.log:Apr 17 18:25:48 129-79-9-73.dhcp-bl.indiana.edu packaging[1284]: Adding Package GeoIP-data-1.5.0-11.el7.noarch in mode u
/var/log/anaconda/journal.log:Apr 17 18:30:08 129-79-9-73.dhcp-bl.indiana.edu packaging[1284]: Installing GeoIP-data (350/993)
/var/log/anaconda/journal.log:Apr 17 18:52:49 129-79-9-73.dhcp-bl.indiana.edu packaging[1284]: GeoIP-data-1.5.0-11.el7.noarch (350/993)
/var/log/anaconda/packaging.log:18:23:24,826 DEBUG yum.verbose.YumBase: TSINFO: Marking GeoIP-data-1.5.0-11.el7.noarch as install for libperfsonar-perl-4.0-1.el7.centos.noarch
/var/log/anaconda/packaging.log:14:25:39,661 DEBUG yum.verbose.YumBase: TSINFO: Marking GeoIP-data-1.5.0-11.el7.noarch as install for libperfsonar-perl-4.0-1.el7.centos.noarch
/var/log/anaconda/packaging.log:14:52:49,525 INFO packaging: GeoIP-data-1.5.0-11.el7.noarch (350/993)
```

**Comment by apertome** [comment]
I'm not really concerned about this for this GeoIP issue -- I already have a workaround  -- but if this package isn't installing correctly, I imagine there are probably others with a similar problem.

**Comment by apertome** [comment]
Fixed in perl-shared@fa00a6c7f81468d112ffa8b8c8d73b755ae106e3. @arlake228 let me know if you think we need a new issue for the ISO RPM issue. Otherwise this is done. I am however going to do a little cleanup on this library on this ticket before closing.

**Comment by apertome** [comment]
Done.
